tests/testthat/helper-write_parameter_file.R

```diff
@@ -20,15 +20,26 @@ write_sample_parameters_file <- function(value,
   )
 
   con <- DBI::dbConnect(duckdb::duckdb())
-  on.exit(DBI::dbDisconnect(con))
+  on.exit(DBI::dbDisconnect(con), add = TRUE)
 
   duckdb::duckdb_register(con, "test_table", df)
   # This is bad practice but `dbBind()` doesn't allow us to parameterize COPY
   # ... TO.  The danger of doing it this way seems quite low risk because it's
   # an ephemeral from a temporary in-memory DB. There's no actual database to
   # guard against a SQL injection attack.
   query <- paste0("COPY (SELECT * FROM test_table) TO '", path, "'")
-  DBI::dbExecute(con, query)
+
+  # Retry a few times because DuckDB throws std::exception intermittently.
```

Comment:
Make issue with this?

Reply:
#97

Reply:
Issue on trying to fix this in a non-sledgehammer way / more robust way